package.json

```diff
@@ -24,10 +24,7 @@
   "devDependencies": {
     "@types/jest": "^24.0.25",
     "@types/node": "^13.1.5",
-    "@types/shelljs": "^0.8.6",
     "jest": "^24.9.0",
-    "nock": "^11.7.2",
```

Comment:
leave this for now because there's some more tests I was going to add that need nock.

Reply:
Added `nock` back in